typings/index.d.ts

```diff
@@ -0,0 +1,93 @@
+declare module 'reduxful' {
+  // fetchUtils
+  export interface RequestAdapterOptions {
+    method: string;
+    url: string;
+    headers: { [key: string]: string };
+    withCredentials: boolean;
+    body: any;
+  }
+
+  export type RequestAdapter = (options: RequestAdapterOptions) => Promise<any>;
+
+  export type Fetcher = (url: string, outOpts: Object) => Promise<any>;
+
+  export function makeFetchAdapter(
+    fetcher: Fetcher,
+    defaultOptions?: Object
+  ): RequestAdapter;
+
+  // utils
+  export interface Resource {
+    isLoaded?: boolean;
+    isUpdating?: boolean;
+    hasError?: boolean;
+  }
+
+  export function isLoaded(resource?: Resource): boolean;
+
+  export function isUpdating(resource?: Resource): boolean;
+
+  export function hasError(resource?: Resource): boolean;
+
+  export function getResourceKey(
+    reqName: string,
+    params?: { [key: string]: string | number }
+  ): string;
+
+  // reduxful
+  export type ApiDescription = Object;
```

Comment:
This can be made more specific, like maybe:

```typescript
{
  [name: string]: {
    url: string | (state: any) => string,
    method?: string,
    resourceAlias?: string,
    // ... other props outlined in https://github.com/godaddy/reduxful/blob/master/docs/api.md#RequestDescription
  }
}
```

Reply:
Ohh there is indeed type for this in types.js`.

I was referencing `api.md` doc, and assumed it was just an [Object](https://github.com/godaddy/reduxful/blob/master/docs/api.md#apidescription--object).

![image](https://user-images.githubusercontent.com/16849235/64983448-51ffc280-d875-11e9-917e-374a7033a91e.png)

Will make an update thanks!